tests/native-lib/pass/ptr_read_access.rs

```diff
@@ -1,4 +1,5 @@
-//@only-target-linux
+//@ignore-target-windows
+//@ignore-target-wasm
```

Comment:
I really want something like `@only-target-unix` but that seems to be unsupported. I looked at the other tests and didn't see an obvious replacement, does this seem reasonable?

Reply:
To do this nicely we need https://github.com/oli-obk/ui_test/issues/162

Reply:
I think even that would still be ugly. We'd need ui-test to know what a "unix" target is and recognize that as `only-unix` or so.

For now, "ignore-wasm" + "ignore-windows" is the way to go.